swift/swift_run_eqpy_sa.sh

```diff
@@ -0,0 +1,122 @@
+#! /usr/bin/env bash
+
+set -eu
+
+if [ "$#" -lt 3 ]; then
+  script_name=$(basename $0)
+  echo "Usage: ${script_name} EXPERIMENT_ID SA_PARAMS_FILE SA_CONFIG_FILE CHECKPOINT_FILE"
```

Comment:
Is it possible to define a checkpoint file here? Perhaps this option should be removed.